src/Plugin/migrate/process/ApplyFilters.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_migration\Plugin\migrate\process;
+
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\oe_migration\ValidConfigurableMigrationPluginInterface;
+use Drupal\oe_migration\FilterFormatManager;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\filter\FilterPluginCollection;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\MigrateException;
+use Drupal\migrate\Row;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Applies filters of the given filter format to a string.
+ *
+ * Available configuration keys:
+ *   - filter_format (mandatory): The filter format id of the destination field,
+ *     eg. full_html. If the destination filter format id is declared after the
+ *     field value or if a YAML variable is used to store the field's value, the
+ *     dynamic detection of the destination field's filter format id won't be
+ *     possible.
+ *   - filters_to_apply (optional): A list of filter id's to apply, eg.
```

Comment:
Might worth mentioning that if there are filters specified here, they will be applied in the order they are set in the site's filter format configuration (based on what I see in ApplyFilters::getFiltersToApply())

Reply:
**Status:** Fixed
Good point, since we don't alter the `weight` of the filters. Thanks for the comment.